pom.xml

```diff
@@ -55,12 +55,6 @@
     <version.npm>6.14.4</version.npm>
     <version.org.apache.poi>4.1.2</version.org.apache.poi>
     <version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>1.0.1.Final</version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>
-    <version.org.mockito>3.7.0</version.org.mockito>
-    <version.org.hibernate>5.4.28.Final</version.org.hibernate>
```

Comment:
The hibernate version in optaplanner-bom is incompatible with the hibernate version in Quarkus; the hibernate version here overrides the optaplanner-bom one so it matches the Quarkus version. As such, this property need to stay until it matches.

Reply:
(IIRC, is also the case for mockito)

Reply:
See the linked PR, which actually upgrades both (and a couple of other versions) to the same version Quarkus declares.
https://github.com/kiegroup/optaplanner/pull/1254

Reply:
Build log from Jenkins build suggests wrong hibernate version is still used; is it not taking that PR into consideration?

Reply:
Unfortunately yes; the only CI capable of taking PRs to the other repositories into consideration is the Build Chain, which apparently has its own issues.